src/Pages/ProfilePage.tsx

```diff
@@ -31,14 +56,56 @@ function ProfilePage() {
-            alt="profile picture"
-          />
+          {userImg ? (
+            <img
+              style={{ width: "250px", height: "250px", borderRadius: "50%" }}
+              src={userImg}
+              alt="profile picture"
+            />
+          ) : (
+            <img
+              style={{ width: "250px", height: "250px", borderRadius: "50%" }}
+              src="https://thumbs.dreamstime.com/b/man-profile-cartoon-smiling-round-icon-vector-illustration-graphic-design-135443422.jpg"
+              alt="profile picture"
+            />
+          )}
+          <button
```

Comment:
The button is invisible for me 

<img width="553" alt="Screenshot 2022-12-01 at 09 46 55" src="https://user-images.githubusercontent.com/80100462/205007682-0c019718-f672-43f5-93e1-126aab4e101c.png">